.gitattributes

```diff
@@ -7,4 +7,4 @@
 *.sh text eol=lf
 *.py text eol=lf
 *.bin binary
-*.approved.* binary
+*.approved.* -text
```

Comment:
I think we can remove this line completely
https://github.com/approvals/ApprovalTests.Net#approved-file-artifacts

Some years ago it was necessary because EOL would change and fail tests.